### ISSUE TITLE
Also Implement Highlights in Preview Mode

### ISSUE THREAD

**Issue by chrisgrieser** [nothingislost/obsidian-dynamic-highlights#26]
The Permanent and the Selected String Highlights both also makes a lot of sense in preview mode, especially since dynamic highlights will often be used by people reading their notes and not necessarily editing them.

**Comment by nothingislost** [comment]
Yeah, agreed but unfortunately that's basically writing an entirely new plugin. I'll probably do this but not until the edit mode features stabilize.

**Comment by ryanwwest** [comment]
I'm wondering if the edit mode features are more stabilized at this point now with the mode to CM6 a few months back? I still use this plugin daily and this would be a great addition.

Though more generally, I think Core Obsidian should probably combine Preview/Reading mode and Live Preview into one mode, which would solve this (though probably require work even then).

**Comment by ReaderGuy42** [comment]
Have there been any updates on this? I'd really love to see my highlights in reading mode.

**Comment by jiajiala** [comment]
And many times, such as embedding, forms, 
these are currently adopted reading patternsI can't see my highlights more and more
